index.html

```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
                 <div class="project-image">
                     <img src="./images/project_1.png">
                 </div>
-                <div class="project-title">
+                <div class="project-title pro-title-one">
                     <h2>Tonic</h2>
                 </div>
-                <div class="projectsub-title">
-                    <ul>
+                <div class="projectsub-title pro-sub-one">
+                    <ul class="sub-title-o">
                         <li class="one">Canopy</li>
                         <li>Back end dev</li>
                         <li>2015</li>
                     </ul>
                 </div>
-                <div class="project-description">
+                <div class="project-description pro-desc-one">
                     <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 </div>
-                <div class="project-tag">
-                    <ul>
+                <div class="project-tag pro-tag-one">
+                    <ul class="pro-tag-o">
                         <li>html</li>
                         <li>css</li>
                         <li>javascript</li>
                     </ul>
                 </div>
-                <div class="project-button">
+                <div class="project-button pro-btn-one">
                     <button>See Project</button>
                 </div>
+                <div id="project-detail-all">
+                    <div class="project-title">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="projectsub-title">
+                        <ul>
+                            <li class="one">Canopy</li>
+                            <li>Back end dev</li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="project-tag">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project-button">
+                        <button>See Project</button>
```

Comment:
- [ ] Kindly adjust the background color of the buttons to match the template. 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215290908-4d3d476b-ac6b-444f-97d3-dbd911c90e1e.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215290915-4730741a-3e67-40d2-af52-acab271aa7b1.png)